### PR TITLE
LPS-145507 Forbid translating content from another group

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/translation/translation-api/src/main/java/com/liferay/translation/exception/XLIFFFileException.java [hunk 1]
```diff
@@ -116,6 +116,25 @@ public MustHaveValidId(Throwable throwable) {
 
 	}
 
+	public static class MustHaveValidModel extends XLIFFFileException {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/translation/translation-api/src/main/java/com/liferay/translation/exception/XLIFFFileException.java:119]: I think that a baseline is needed

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**